assets/queries/common/passwords_and_secrets_in_infrastructure_code/metadata.json

```diff
@@ -5,5 +5,10 @@
   "category": "Secret Management",
   "descriptionText": "Query to find passwords and secrets in infrastructure code.",
   "descriptionUrl": "https://kics.io/",
-  "platform": "Common"
+  "platform": "Common",
+  "override": {
+    "override": {
+      "queryName": "teste"
```

Comment:
@felipe-avelar Is this supposed to be like this, or was it for testing purposes?